### ISSUE TITLE
Bullets within step components do not display

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#655]
Bullets within step components do not display _at all_ on certain devices.  
This is a duplicate of issue https://github.com/cityofaustin/techstack/issues/646

Example: [Get a birth or death certificate](http://staging.austin.gov/services/birth-and-death-certificates)

**iPhone 5S, Safari, 8/28/18:**
_Bullets within steps don't display_
![img_ee561224b815-1](https://user-images.githubusercontent.com/1463708/44735784-05531900-aab3-11e8-87b6-3eed07fa081a.jpeg)

**Mac OSX 10.13, Chrome, 8/31/18:**
_Bullets within steps don't display_
![screen shot 2018-08-31 at 2 27 36 pm](https://user-images.githubusercontent.com/1463708/44932196-0d68be00-ad2a-11e8-9713-28faf7f46fc5.png)

**Comment by amenity** [comment]
Marking as dependent on #650 so that we don't fix something only to have to then implement a new design.
